### PR TITLE
Update prices_ethereum_tokens.sql

### PR DESCRIPTION
**Pull request by peterrliem**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1624,7 +1624,17 @@ FROM
...
+    ('gene-genomesdao', 'ethereum', 'GENE', 0x21413c119b0c11c5d96ae1bd328917bc5c8ed67e, 18),
+    ('rbc-rubic', 'ethereum', 'RBC', 0x3330bfb7332ca23cd071631837dc289b09c33333, 18),
+    ('magic-magic-arbitrumk', 'ethereum', 'MAGIC', 0xb0c7a3ba49c7a6eaba6cd4a96c55a1391070ac9a, 18),
+    ('premia-premia', 'ethereum', 'PREMIA', 0x6399c842dd2be3de30bf99bc7d1bbf6fa3650e70, 18),
+    ('stfx-stfx', 'ethereum', 'STFX', 0x9343e24716659a3551eb10aff9472a2dcad5db2d, 18),
+    ('flux-datamine-flux', 'ethereum', 'FLUX', 0x469eda64aed3a3ad6f868c44564291aa415cb1d9, 18),
+    ('vee-zeeverse', 'ethereum', 'VEE', 0x7616113782AaDAB041d7B10d474F8A0c04EFf258, 18),
+    ('noisegpt-noisegpt', 'ethereum', 'NOISEGPT', 0xf23a2BF7619Ad93300499c3fC7d7145A06CcA562, 18),
+    ('tpy-thrupenny', 'ethereum', 'TPY', 0x968cbe62c830a0ccf4381614662398505657a2a9, 8)
+    ('mvd-metavault-dao', 'ethereum', 'MVD', 0x788b6d2b37aa51d916f2837ae25b05f0e61339d1, 9)
```
Comment (Hosuke) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1637]:
```suggestion
    ('tpy-thrupenny', 'ethereum', 'TPY', 0x968cbe62c830a0ccf4381614662398505657a2a9, 8),
    ('mvd-metavault-dao', 'ethereum', 'MVD', 0x788b6d2b37aa51d916f2837ae25b05f0e61339d1, 9)
```

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 2]
```diff
@@ -1624,7 +1624,17 @@ FROM
     ('beam-beam-eth', 'ethereum', 'BEAM', 0x62d0a8458ed7719fdaf978fe5929c6d342b0bfce, 18),
     ('sdl-stakelink', 'ethereum', 'SDL', 0xa95c5ebb86e0de73b4fb8c47a45b792cfea28c23, 18),
     ('ton-tokamak-network', 'ethereum', 'TON', 0x2be5e8c109e2197D077D13A82dAead6a9b3433C5, 18),
-    ('tos-tonstarter', 'ethereum', 'TOS', 0x409c4d8cd5d2924b9bc5509230d16a61289c8153, 18)
+    ('tos-tonstarter', 'ethereum', 'TOS', 0x409c4d8cd5d2924b9bc5509230d16a61289c8153, 18),
+    ('gene-genomesdao', 'ethereum', 'GENE', 0x21413c119b0c11c5d96ae1bd328917bc5c8ed67e, 18),
+    ('rbc-rubic', 'ethereum', 'RBC', 0x3330bfb7332ca23cd071631837dc289b09c33333, 18),
+    ('magic-magic-arbitrumk', 'ethereum', 'MAGIC', 0xb0c7a3ba49c7a6eaba6cd4a96c55a1391070ac9a, 18),
```
Comment (Hosuke) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1630]:
```suggestion
    ('magic-magic-arbitrum', 'ethereum', 'MAGIC', 0xb0c7a3ba49c7a6eaba6cd4a96c55a1391070ac9a, 18),
```

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 3]
```diff
@@ -1624,7 +1624,17 @@ FROM
...
+    ('gene-genomesdao', 'ethereum', 'GENE', 0x21413c119b0c11c5d96ae1bd328917bc5c8ed67e, 18),
+    ('rbc-rubic', 'ethereum', 'RBC', 0x3330bfb7332ca23cd071631837dc289b09c33333, 18),
+    ('magic-magic-arbitrum', 'ethereum', 'MAGIC', 0xb0c7a3ba49c7a6eaba6cd4a96c55a1391070ac9a, 18),
+    ('premia-premia', 'ethereum', 'PREMIA', 0x6399c842dd2be3de30bf99bc7d1bbf6fa3650e70, 18),
+    ('stfx-stfx', 'ethereum', 'STFX', 0x9343e24716659a3551eb10aff9472a2dcad5db2d, 18),
+    ('flux-datamine-flux', 'ethereum', 'FLUX', 0x469eda64aed3a3ad6f868c44564291aa415cb1d9, 18),
+    ('vee-zeeverse', 'ethereum', 'VEE', 0x7616113782AaDAB041d7B10d474F8A0c04EFf258, 18),
+    ('noisegpt-noisegpt', 'ethereum', 'NOISEGPT', 0xf23a2BF7619Ad93300499c3fC7d7145A06CcA562, 18),
+    ('tpy-thrupenny', 'ethereum', 'TPY', 0x968cbe62c830a0ccf4381614662398505657a2a9, 8),
+    ('mvd-metavault-dao', 'ethereum', 'MVD', 0x788b6d2b37aa51d916f2837ae25b05f0e61339d1, 9)
```
Comment (Hosuke) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1637]: There are some inactive(price feeds) tokens here, we may need to remove them:
https://github.com/duneanalytics/spellbook/actions/runs/7476909468/job/20348390637?pr=5136#step:6:42

**Review by Hosuke — CHANGES_REQUESTED** [review]

**Comment by peterrliem** [comment]
Let's get them removed then.

Thank you!

**Review by jeff-dude — APPROVED** [review]
should be good now, we can ignore the symbol error as it looks like coinpaprika is incorrect

**Pull request merged**